### ISSUE TITLE
[DUPLICATE]bug: White Background overlays view on Keyboard show / hide on Android #4057

### ISSUE THREAD

**Issue by rahul202** [driftyco/ionic#8122]
Actually i saw issue no.4057 status closed.

Hence opening a  new one, if that issue can be reopened, please close this open a new one.

I have a ng-repeat of list of 100 + items to display.
Step1:
Have a list of 100+ items in ng-repeat. inside ion-content
Step2. Focus ur input box.
Step3: Keyboard pops up. Enter ur text.(Controller process ur text, and keyboard pulled down).
Step4: When keyboard is pulled down. <ion-content> should scoll down immediately, but it seems there is delay of near to 1-2 sec.

Android version:6.0.1
Mobile model: Moto g3 turbo

Basically, issue seems to be:
Ion-content doesnt scolls back to its previous position immediately, when keyboard closes. 
Time difference between. keyboard close and Ion-content scrolling back to full screen or previous position is quite observable say 1-2 sec.

**Comment by chocolei** [comment]
In AndroidManifest.xml, set 
`android:windowSoftInputMode="adjustPan"`
instead of adjustSize work perfect for me.

ref:
[http://stackoverflow.com/questions/35201412/android-white-background-when-keyboard-fades-away](url)

**Comment by rahul202** [comment]
Guess, bug was cause of large number of list items and images which i was trying to display...., if i reduce the number of items for displaying, it works fine....

**Comment by jgw96** [comment]
Hello, thanks for using Ionic! After reviewing this it definitely seems like the delay is caused by the webview taking a second to render a huge list. Mobile applications (including native applications) generally should not use long lists as its generally not good ux and can cause performance issues (as this issue shows). I would recommend using `collection-repeat` to do lazy loading of data for your list. Thanks!
